patches/chrome-browser-ui-webui-downloads-downloads_ui.cc.patch

```diff
@@ -6,7 +6,7 @@ index 0c6e7cb046955ba2619f2898bc09cc9cbe3431c9..74958f8f23c246a771761f71abc1b46c
  content::WebUIDataSource* CreateDownloadsUIHTMLSource(Profile* profile) {
    content::WebUIDataSource* source =
        content::WebUIDataSource::Create(chrome::kChromeUIDownloadsHost);
-+  BraveCustomizeDownloadsDataSource(source);
++  BRAVE_CREATE_DOWNLOADS_UI_HTML_SOURCE
    webui::SetupWebUIDataSource(
```

Comment:
re: patches below, maybe we should actually just override `SetupWebUIDataSource` and use that to insert our resources in any webui based on the resource_id

Reply:
Created https://github.com/brave/brave-browser/issues/25461 to address the above in a follow up.